src/one_dragon_qt/widgets/install_card/code_install_card.py

```
@@ -31,22 +35,32 @@ def __init__(self, ctx: OneDragonEnvContext, parent=None):
             self.check_and_update_display()
             self.updated = True
-        else:
-            msg = msg + ' 请检查网络连接，必要时切换网络环境（例如使用手机热点）后重试'
+        elif self._last_sync_status is GitSyncStatus.RUNTIME_INCOMPATIBLE:
+            msg = f'{msg} 请更新集成启动器后重试'
+            self.update_display(FluentIcon.INFO.icon(color=FluentThemeColor.GOLD.value), gt(msg))
+        elif self._last_sync_status is GitSyncStatus.BUILTIN_TAG_UNAVAILABLE:
+            msg = f'{msg} 请更新集成启动器或使用对应版本的安装包后重试'
+            self.update_display(FluentIcon.INFO.icon(color=FluentThemeColor.GOLD.value), gt(msg))
+        elif self._last_sync_status is GitSyncStatus.FAILED:
+            msg = f'{msg} 请检查网络连接，必要时切换网络环境（例如使用手机热点）后重试'
```

Ruff (0.16.0)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)